user.js

```diff
@@ -567,7 +567,7 @@ user_pref("dom.disable_window_move_resize", true);
 /*** [SECTION 2600]: MISCELLANEOUS ***/
 user_pref("_user.js.parrot", "2600 syntax error: the parrot's run down the curtain!");
 /* 2601: prevent accessibility services from accessing your browser [RESTART]
- * [1] https://support.mozilla.org/kb/accessibility-services ***/
+ * [1] https://support.mozilla.org/en-US/kb/accessibility-services ***/
```

Comment:
Let's leave language choice up to Mozilla Web servers please :slightly_smiling_face: